docs/cli/teams/commands.md

```diff
@@ -2,25 +2,93 @@
 title: Commands
 description: Use these Amplify CLI commands to manage a team workflow with multiple environments.
 ---
-
-* amplify env add <br>
-Adds a new environment to your Amplify Project
-* amplify env list [--details] [--json] <br>
-Displays a list of all the environments in your Amplify project
-* amplify env remove \<env-name\> <br>
-Removes an environment from the Amplify project
-* amplify env get --name \<env-name\> <br>
-Displays the details of the environment specified in the command 
-* amplify env pull <br>
-Pulls your environment from the cloud without impacting any local backend edits. Add the `--restore` flag to overwrite your local backend edits  (operates like the `amplify pull` command).
-* amplify env import<br>
-Imports an already existing Amplify project environment stack to your local backend. Here's a sample usage of the same
-
+ 
+## Team workflows
+ 
+Amplify environments help you manage your local and cloud environments to mimic your team workflows. Common tasks include:
+- Manage environments to support development processes. Eg: development, staging, production
+- Test new features safely
+- Share environments between team members
+- Support team workflows
+ 
+To display all commands available for a new Amplify project, run the following command from the root directory.
+```
+amplify env
+```
+ 
+## Commands overview
+ 
+| Command  | Description  |
+|---|---|
+| `amplify env add` | Adds a new environment. [More](#add-a-new-environment) |
+| `amplify env pull [--restore]` | Pulls the current environment from the cloud. [More](#pull-the-environment-from-the-cloud) |
```

Comment:
Instead of the `more` can we put the links around the commands themselves? If that's not possible, let's do "Learn more".